Sources/GraphQLLanguage/Builder.swift

```diff
@@ -10,24 +10,36 @@ import Foundation
 
 enum BuildError: Error, CustomDebugStringConvertible {
     case unwrap
-    case unexpectedContext(_ context: ParserRuleContext)
+    case unexpectedContext(text: String, token: Token?)
 
     // TODO: Improve error description
     var debugDescription: String {
         switch self {
         case .unwrap:
             return "Required field must not be nil. Probably a parser bug."
-        case .unexpectedContext(let context):
-            let rule = context.toStringTree(GraphQLParser.ruleNames)
-            if let token = context.getStart() {
-                let line = token.getLine()
-                let position = token.getCharPositionInLine()
-                return "Build error at: \(rule) line:\(line) position:\(position)"
-            } else {
-                return "Build error at: \(rule)"
-            }
+        case .unexpectedContext(let text, let token):
+            return "Build error: \(text)" + (token.map { token in
+                " at line:\(token.getLine()) position:\(token.getCharPositionInLine())"
+            } ?? "")
```

Comment:
I think `Token` is a protocol and in the implementation, it still can reference value from other object and so likely capture line and position would be better. So, I suggest adding each as field of enum then
```
case unexpectedContext(text: String, line: Int?, position: Int?)
...
switch self {
    ...
    case .unexpectedContext(let text, .some(let line), .some(let position)):
        return "Build error: \(text) at line: \(line) position: \(position)"
    case .unexpectedContext(let text, _, _):
        return "Build error: \(text)"
}
```
and in the following helper function,
```
...
return .unexpectedContext(text: rule, line: token?.getLine(), position: token?.getCharPositionInLine())
```

Reply:
> it still can reference value from other object

I was unaware of this - thanks for the feedback! I made the suggested change. (And also renamed `text` to `rule` since its content felt ambiguous to me).